provisioner/inspec/provisioner.go

```diff
@@ -157,6 +159,10 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 		errs = packersdk.MultiErrorAppend(errs, fmt.Errorf("user: could not determine current user from environment."))
 	}
 
+	if p.config.ValidExitCodes == nil {
+		p.config.ValidExitCodes = []int{0, 101}
```

Comment:
https://docs.chef.io/inspec/cli/#exec leaving here some docs of why 101. 
`101  normal exit, at least one test skipped but none failed`